protos/guard/Guard.proto

```diff
@@ -0,0 +1,192 @@
+syntax = "proto3";
```

Comment:
Filename should be lowercase `guard.proto` ... I thought linter should do that. Weiyu is taking a look at linter again.

Reply:
Solved.

Reply:
Still is upper case Guard.proto, maybe needs a git push?